packages/eset_protect/data_stream/event/elasticsearch/ingest_pipeline/default.yml

```
@@ -15,10 +15,33 @@ processors:
       field: event.original
       tag: json_event_original
       target_field: json
+      if: "ctx.event?.original.startsWith(\"{\\\"\")"
       on_failure:
         - append:
             field: error.message
             value: "Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}"
+  - grok:
+      field: event.original
+      tag: grok_event_original
+      patterns:
+      - 'Detection type: %{DATA:json.threat_name} Detection name: %{DATA:json.threat_name}
+        Computer name: %{HOSTNAME:json.hostname} Logged user: %{DATA:user.domain}\\%{DATA:json.user}
+        Time of occurrence: %{ESET_TIMESTAMP:json.occured_plaintext} Scanner: %{DATA:json.scanner_id}
+        Action performed: %{DATA:json.action_taken}$'
+      - '^%{DATA:message}  %{HOSTNAME:json.hostname} %{ESET_TIMESTAMP:json.occured_plaintext} %{DATA:eset_protect.event.threat_type} %{NOTSPACE:threat.indicator.name} %{ESET_THREAT_INDICATOR_TYPE:json.object_type} %{GREEDYDATA:json.object_uri}$'
```

```suggestion
      - '^%{DATA:message} %{HOSTNAME:json.hostname} %{ESET_TIMESTAMP:json.occured_plaintext} %{DATA:eset_protect.event.threat_type} %{NOTSPACE:threat.indicator.name} %{ESET_THREAT_INDICATOR_TYPE:json.object_type} %{GREEDYDATA:json.object_uri}$'
```
Just checking if this double space is necessary. I see the event also has double space, but in case we aren't sure we could go for:
`%{DATA:message}\s+%{HOSTNAME:json.hostname}`